packages/wonder-blocks-core/components/text.js

```diff
@@ -30,6 +30,19 @@ const styles = StyleSheet.create({
     },
 });
 
+/**
+ * Text is a building block for constructing other components. `View` roughly
+ * maps to `div` and `Text` roughly maps to `span`. You can override which tag
+ * is used to render the component (for semantic purposes) by specifying the
+ * `tag` prop.
```

Comment:
The `View` parts of this comment could be removed.  I think the docs for `Text` and `View` need improvement so this is something that can be addressed whenever we improve the docs for both of these components.  As I was reading through the docs for "Core" it would be nice if we could categorize exports into components, functions, constants.  But that would be a bigger general docs improvement change.

Reply:
Removed the View part.